packages/modules/create-itwin-viewer/extensions/web/intermediate.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "../../web-viewer-react/node_modules/@itwin/web-viewer-react/src";
```

Comment:
all these providers should be imported from "@itwin/web-viewer-react" or "@itwin/desktop-viewer-react"

also, there is no need for the "StandardNavigationToolsProvider", we have the "ViewerNavigationToolsProvider" which just wraps those